src/PerfView/Authentication.cs

```diff
@@ -214,6 +214,9 @@ public static void Configure(this SymbolReaderHttpHandler handler, Authenticatio
 
             handler.ClearHandlers();
 
+            // Always add Symweb authentication.
+            handler.AddSymwebAuthentication(log);
```

Comment:
PerfView doesn't do anything to prevent, this, but users would have to login.  This will come in the form of an authentication prompt.  If the user doesn't specify the new symweb in their symbol path, then they won't ever hit this.  Additionally, we don't specify the new symweb in the default symbol path, so users will need to specify it.  I intend to send mail to the internal users group alias on how to specify the new symweb in the symbol path once this is released.  Does that address your concern?

Reply:
I see. Thanks.